packages/clerk-js/src/core/resources/SamlAccount.ts

```diff
@@ -27,7 +28,7 @@ export class SamlAccount extends BaseResource implements SamlAccountResource {
     }
 
     this.id = data.id;
-    this.provider = data.provider;
+    this.provider = data.provider as SamlIdpSlug;
```

Comment:
❓ Instead of using the keyword `as`, shouldn't we update the `SamlAccountJSON` type and define the actual type of provider field to be SamlIdpSlug?

Reply:
We could, I was hesitant to move a custom type into the JSON payload, but I guess there should be no issue.

Reply:
Moved to `SamlAccountJSON`